actor/src/lib.rs

```diff
@@ -498,18 +504,8 @@ impl<A: Actor> Context<A> {
         self.addr().recipient()
     }
 
-    /// Sends actor specified message
-    pub fn notify<M>(&self, message: M)
-    where
-        M: Message<Result = ()> + Send + 'static,
-        A: ContextHandler<M>,
-    {
-        let addr = self.addr();
-        task::spawn(async move { addr.do_send(message).await }.in_current_span());
-    }
-
     /// Sends actor specified message in some time
-    pub fn notify_later<M>(&self, message: M, later: Duration)
+    pub fn notify<M>(&self, message: M, later: Duration)
```

Comment:
remove the reference to `notify_later` or `notify` in the docs then? Which ever name you choose